test/decl/operator/power_operator_imported.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -I %S/Inputs/custom-modules -I %t %s -verify
+
+// REQUIRES: objc_interop
+
+import PowerOperator
+
+func testInvalidPowerOperationOutOfScope() {
+  func a(_ value: Double) { }
+  let x: Double = 3.0
+  let y: Double = 3.0
+  let z: Double = x**y // expected-error {{no operator '**' is defined; did you mean 'pow(_:_:)'?}}
+  let w: Double = a(x**2.0) // expected-error {{no operator '**' is defined; did you mean 'pow(_:_:)'?}}
```

Comment:
Can you add one more test case where there is no `pow` in scope, and in that situation check that we are not emitting the specialized diagnostic? I'm not sure if `-verify` has a way to do it, but you can pipe the output to FileCheck to do so.

Maybe it can be part of this same file but with 2 RUN lines `-DWITH_POW`, `-DWITHOUT_POW` or something.

Reply:
New test in `operator.swift` test does that where no `pow` is in scope and only generic `cannot find operator '**' in scope` error is emitted.